### ISSUE TITLE
Incorrect element is used to retrieve FormController instance in ngSubmitDecorator.js

### ISSUE THREAD

**Issue by hinok** [jonsamwell/angular-auto-validate#73]
I think you should use element object from post-link function, not from compile.

https://github.com/jonsamwell/angular-auto-validate/blob/master/src/config/ngSubmitDecorator.js

 **18 line**:
```
var formController = $element.controller('form');
```

should be changed to
```
var formController = element.controller('form');
```

I have weird problem when formController was always undefined in one case.
I changed it in my fork: https://github.com/hinok/angular-auto-validate/commit/a6deac7e5053923208095933f06765034cfbebfe

After that change all tests still passed and issue was solved.

**Comment by jonsamwell** [comment]
Could you do a pull request?

**Comment by hinok** [comment]
No problem but my fork has a few more changes which are related to  https://github.com/jonsamwell/angular-auto-validate/issues/66

I can make a pull request but it will contain a lot of more changes in code than just this one or I will fork the repo again and make just this one change and then make a pull request.

**Comment by jonsamwell** [comment]
Fixed in next release (V1.18.7)
